scripts/windows/msys2_env_build.sh

```diff
@@ -66,3 +67,4 @@ INCLUDE="${INCLUDE};${WIN_MSYS2_ROOT}\\mingw64\\include;${WIN_MSYS2_ROOT}\\usr\\
 LIB="${LIB};${WIN_MSYS2_ROOT}\\mingw64\\lib;${WIN_MSYS2_ROOT}\\mingw64\\bin;${WIN_ERL_TOP}\\usr\\lib;"
 
 export INCLUDE LIB PATH ERL_TOP WIN_ERL_TOP COMSPEC
+export INCLUDE LIB PATH ERL_TOP WIN_ERL_TOP COMSPEC CPPFLAGS
```

Comment:
not sure why this line is duplicate of the above, but it should be harmless and cleaned up later

Reply:
well the PR wasn't ready :)